dolphin/alert_manager/alert_processor.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2020 The SODA Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from oslo_log import log
+
+from dolphin.drivers import manager as driver_manager
+from dolphin import db
+
+LOG = log.getLogger(__name__)
+
+
+class AlertProcessor(object):
+    """Alert model translation and export functions"""
+    def __init__(self):
+        self.driver_manager = driver_manager.DriverManager()
+
+    def process_alert_info(self, alert):
+        """Fills alert model using driver manager interface."""
+
+        # Get storage_id using source ip address
+        # If device wants to use source ip different from registration ip, it can be done as part of alert source config
```

Comment:
remove unrelated comments and format the comments.

Reply:
Updated